api/src/opentrons/protocol_api/labware.py

```diff
@@ -1061,7 +1066,10 @@ def split_tipracks(tip_racks: List[Labware]) -> Tuple[Labware, List[Labware]]:
 
 # TODO(mc, 2022-11-09): implementation detail, move to core
 def select_tiprack_from_list(
-    tip_racks: List[Labware], num_channels: int, starting_point: Optional[Well] = None
+    tip_racks: List[Labware],
+    num_channels: int,
+    starting_point: Optional[Well] = None,
+    nozzle_map: Optional[NozzleMap] = None,
```

Comment:
Ditto other comment about making this new argument keyword-only. (I'm conservatively assuming this function is just as public, but I haven't checked and it might not be.)

```suggestion
    tip_racks: List[Labware],
    num_channels: int,
    starting_point: Optional[Well] = None,
    *,
    nozzle_map: Optional[NozzleMap] = None,
```